readme.md

```diff
@@ -50,7 +50,7 @@ Recursively sort keys.
 
 ##### compare
 
-Type: `Function`
+Type: `function`
```

Comment:
This one should actually be uppercase. I'm following TS types syntax. https://github.com/sindresorhus/meta/issues/1#issuecomment-483966935